win32ss/include/ntuser.h

```diff
@@ -3365,6 +3365,9 @@ NtUserSetWindowLongPtr(
     DWORD Index,
     LONG_PTR NewValue,
     BOOL Ansi);
+#else
+#define NtUserSetWindowLongPtr(hWnd, Index, NewValue, Ansi) \
+    ((LONG_PTR)NtUserSetWindowLong((hWnd), (Index), (LONG)(NewValue), (Ansi)))
```

Comment:
I think that in 32-bits it is sufficient to do
```suggestion
#define NtUserSetWindowLongPtr  NtUserSetWindowLong
```
(as it is done in user-mode for Get/SetWindowLongPtr), since the `LONG_PTR` type will automatically become `LONG`.

Reply:
OK.